rust/numaflow-core/src/message.rs

```diff
@@ -103,7 +79,7 @@ impl TryFrom<async_nats::Message> for Message {
         let event_time = Utc::now();
         let offset = None;
         let id = MessageID {
-            vertex_name: get_vertex_name().to_string(),
+            vertex_name: config::get_vertex_name().to_string(),
```

Comment:
Having `get_vertex_name()` in this `try_from` function is weird. Do we have to implement `try_from`, or a until function to convert Message (with vertexName in the signature) also works?

Reply:
what do you mean by an `until` function?